src/hotspot/cpu/aarch64/assembler_aarch64.cpp

```diff
@@ -187,6 +187,18 @@ void Address::lea(MacroAssembler *as, Register r) const {
     zrf(Rd, 0);
   }
 
+void Assembler::prfm(const Address &adr, prfop pfop) {
```

Comment:
```suggestion
// This encoding is similar (but not quite identical) to the encoding used
// by literal ld/st. see JDK-8324123.
void Assembler::prfm(const Address &adr, prfop pfop) {
```

Reply:
Thanks for your suggestions. I have added the comments, please check